cluster-diagnosis/__main__.py

```diff
@@ -45,20 +46,25 @@
         subparsers = parser.add_subparsers(dest='sysdump')
         subparsers.required = False
         parser_sysdump = subparsers.add_parser('sysdump',
-                                               help='collect logs and other '
-                                                    'useful information')
+                                               help='Collect logs and other '
+                                                    'useful information.')
         parser_sysdump.add_argument('--since',
                                     help='Only return logs newer than a '
                                          'relative duration like 5s, 2m, or'
-                                         ' 3h. Defaults to all logs.',
-                                    default='12h')
+                                         ' 3h. Defaults to 30m.',
+                                    default='30m')
         parser_sysdump.add_argument('--size-limit', type=int,
                                     help='size limit (bytes) for the '
-                                         'collected logs',
-                                    default=256 * 1024 * 1024)
+                                         'collected logs. '
+                                         'Defaults to 1048576 (1MB).',
+                                    default=1 * 1024 * 1024)
         parser_sysdump.add_argument('--output',
                                     help='Output filename without '
                                          ' .zip extension')
+        parser_sysdump.add_argument('--quick', type=distutils.util.strtobool,
+                                    default="false",
+                                    help='Enable quick mode. '
```

Comment:
This is a tautological definition; it'd be helpful to explicitly note that logs and bugtool output are not collected with this enabled.

Reply:
Fixed.